src/common/hexchat.c

```diff
@@ -403,10 +403,15 @@ irc_init (session *sess)
 
 	if (prefs.hex_notify_timeout)
 		notify_tag = fe_timeout_add (prefs.hex_notify_timeout * 1000,
-											  notify_checklist, 0);
+					     notify_checklist, NULL);
 
-	fe_timeout_add (prefs.hex_away_timeout * 1000, away_check, 0);
-	fe_timeout_add (500, hexchat_misc_checks, 0);
+	fe_timeout_add (prefs.hex_away_timeout * 1000, away_check, NULL);
+	fe_timeout_add (1000, hexchat_check_dcc, NULL);
+	if (prefs.hex_gui_lagometer)
```

Comment:
TingPing <notifications@github.com> writes:

>>  
>> -	fe_timeout_add (prefs.hex_away_timeout * 1000, away_check, 0);
>> -	fe_timeout_add (500, hexchat_misc_checks, 0);
>> +	fe_timeout_add (prefs.hex_away_timeout * 1000, away_check, NULL);
>> +	fe_timeout_add (1000, hexchat_check_dcc, NULL);
>> +	if (prefs.hex_gui_lagometer)
>
> This means you have to restart the client to enable that feature, less than ideal.
>
Indeed; I've pushed one possible fix for this.